.travis.yml

```diff
@@ -14,6 +14,14 @@ matrix:
     - php: 7.1
       env:
         - DEPENDENCIES="--prefer-lowest --prefer-stable"
+    - php: 7.2
+      env:
+        - DEPENDENCIES=""
+        - EXECUTE_CS_CHECK=true
```

Comment:
remove this line please, EXECUTE_CS_CHECK is meant only to be executed once per build

Reply:
Done